proto/c1/connector/v2/action.proto

```
@@ -0,0 +1,76 @@
+option go_package = "github.com/conductorone/baton-sdk/pb/c1/config/v1";
+
+
+service ActionService {
+  rpc Invoke(ActionServiceInvokeRequest) returns (ActionServiceInvokeResponse);
+  rpc Status(ActionServiceStatusRequest) returns (ActionServiceStatusResponse);
+  rpc GetSchema(ActionServiceGetSchemaRequest) returns (ActionServiceGetSchemaResponse);
+  rpc ListSchemas(ActionServiceListSchemasRequest) returns (ActionServiceListSchemasResponse);
+}
+
+enum BatonActionStatus {
+  STATUS_UNKNOWN = 0;
```

Buf (1.47.2)

Enum value name "STATUS_UNKNOWN" should be prefixed with "BATON_ACTION_STATUS_".

(ENUM_VALUE_PREFIX)